roles/edpm_nova/meta/argument_specs.yml

```diff
@@ -25,6 +25,32 @@ argument_specs:
         description: |
           The path to the directory where the nova config files
           will be rendered on the compute node.
+      edpm_nova_certs_src:
+        type: str
+        default: /var/lib/openstack/certs
+        description: |
+          The path to the directory containing the nova cert and key files
+          in the ansibleEE container. This is the directory
+          where all TLS certs and keys for the nova service are mounted.
+      edpm_nova_certs_dest:
+        type: str
+        default: /var/lib/openstack/certs/nova
+        description: |
+          The path to the directory where the nova cert and key files
+          will be rendered on the compute node.
+      edpm_nova_cacerts_src:
+        type: str
+        default: /var/lib/openstack/cacerts
+        description: |
+          The path to the directory containing the cacert files
+          in the ansibleEE container. This is the directory
+          where all cacert PEM files for the nova service are mounted.
+      edpm_nova_cacerts_dest:
+        type: str
+        default: /var/lib/openstack/cacerts/nova
+        description: |
+          The path to the directory where the cacert PEM files
+          for the nova service will be rendered on the compute node.
```

Comment:
i would have expected edpm_nova_tls_certs_enabled to be required here too.

Reply:
done